### DISCUSSION TITLE
Slack notification preview problem. How to send text message.

### DISCUSSION THREAD

**Discussion by kartal-huseyin** [test-results-reporter/reporter#125] [Q&A] [answered]
Since our report includes just inputs as an attachment. I think we requrie to send a text with the report. How to put a text in to report.

![image](https://user-images.githubusercontent.com/98432117/228186281-b845f354-e95b-4727-9e5d-2aa4db8e1ebf.png)

**Comment by ASaiAnudeep** [comment]
Hello @kartal-huseyin , can you please provide us more details.

**Comment by ASaiAnudeep** [comment, marked as answer]
@kartal-huseyin the new format will address the preview problem in slack - https://github.com/test-results-reporter/testbeats/pull/272

```json
{
  "name": "slack",
  "condition": "fail",
  "inputs": {
    "url": "<slack-incoming-webhook-url>",
    "message_format": "blocks"
  }
}

```
